bundles/ch.elexis.core/src/ch/elexis/core/types/LocalizeUtil.java

```diff
@@ -1,29 +1,69 @@
-package ch.elexis.core.types;
-
-import java.util.ResourceBundle;
-
-import org.eclipse.emf.common.util.Enumerator;
-
-import ch.elexis.core.interfaces.ILocalizedEnum;
-import ch.elexis.core.model.MaritalStatus;
-
-public class LocalizeUtil {
-	/**
-	 * Use with EMF generated not instances of {@link ILocalizedEnum}. Get the localized String for

```

Comment:
I prefer implementing ILocalizedEnum for interfaces where a localization is relevant.